testing/events.go

```diff
@@ -75,13 +86,13 @@ func ParsePacketFromEvents(events []abci.Event) (channeltypes.Packet, error) {
 // ParsePacketsFromEvents parses events emitted from a MsgRecvPacket and returns
 // all the packets found.
 // Returns an error if no packet is found.
-func ParsePacketsFromEvents(events []abci.Event) ([]channeltypes.Packet, error) {
+func ParsePacketsFromEvents(eventType string, events []abci.Event) ([]channeltypes.Packet, error) {
 	ferr := func(err error) ([]channeltypes.Packet, error) {
 		return nil, fmt.Errorf("ibctesting.ParsePacketsFromEvents: %w", err)
 	}
 	var packets []channeltypes.Packet
 	for _, ev := range events {
-		if ev.Type == channeltypes.EventTypeSendPacket {
```

Comment:
couldn't we have just done logical or `||` on event type `channeltypes.EventTypeSendPacket || channeltypes.EventTypeRecvPacket`?

Reply:
shouldn't the `result.Events` from a recv packet which does a send internally include send packet events which are parsable here too?